hypervisor/include/arch/x86/idt.h

```diff
@@ -74,6 +68,15 @@ struct host_idt_descriptor {
 extern struct host_idt HOST_IDT;
 extern struct host_idt_descriptor HOST_IDTR;
 
+#else /* ASSEMBLER */
+
+/* Interrupt Descriptor Table (LDT) selectors are 16 bytes on x86-64 instead of 8 bytes. */
+#define     X64_IDT_DESC_SIZE   (0x10)
+/* Number of the HOST IDT entries */
+#define     HOST_IDT_ENTRIES    (0x100)
```

Comment:
This will lead to complain in irq.c

Reply:
For irq.c, it will use the following MACRO definition.

#ifndef ASSEMBLER

/* Number of the HOST IDT entries */
#define     HOST_IDT_ENTRIES    (0x100U)

Reply:
How about to use (unsigned int) HOST_IDT_ENTRIES ?

Reply:
I talked with Junjie about this option. He said that it is not preferred.
If HOST_IDT_ENTRIES is used multiple times in C source code, the cast would need to be done multiple times.